core/src/desul/atomics/Compare_Exchange_GCC.hpp

```diff
@@ -18,72 +18,71 @@ SPDX-License-Identifier: (BSD-3-Clause)
 namespace desul {
 
 namespace Impl {
-template<class T>
+template <class T>
 struct atomic_exchange_available_gcc {
   constexpr static bool value =
 #ifndef DESUL_HAVE_LIBATOMIC
-    ((sizeof(T)==4 && alignof(T)==4) ||
+      ((sizeof(T) == 4 && alignof(T) == 4) ||
 #ifdef DESUL_HAVE_16BYTE_COMPARE_AND_SWAP
-     (sizeof(T)==16 && alignof(T)==16) ||
+       (sizeof(T) == 16 && alignof(T) == 16) ||
 #endif
-     (sizeof(T)==8 && alignof(T)==8)) &&
+       (sizeof(T) == 8 && alignof(T) == 8)) &&
 #endif
-    std::is_trivially_copyable<T>::value;
+      std::is_trivially_copyable<T>::value;
 };
-} //namespace Impl
+}  // namespace Impl
 
-#if defined(__clang__) && (__clang_major__>=7) && !defined(__APPLE__)
+#if defined(__clang__) && (__clang_major__ >= 7) && !defined(__APPLE__)
 // Disable warning for large atomics on clang 7 and up (checked with godbolt)
-// error: large atomic operation may incur significant performance penalty [-Werror,-Watomic-alignment]
-// https://godbolt.org/z/G7YhqhbG6
+// error: large atomic operation may incur significant performance penalty
+// [-Werror,-Watomic-alignment] https://godbolt.org/z/G7YhqhbG6
```

Comment:
```suggestion
// clang-format off
// error: large atomic operation may incur significant performance penalty [-Werror,-Watomic-alignment]
// clang-format on
// https://godbolt.org/z/G7YhqhbG6
```
Uggg.  Did we merge that into Desul?

Reply:
yeah